torch/distributed/distributed_c10d.py

```diff
@@ -1526,6 +1551,83 @@ def _set_pg_timeout(timeout: timedelta, group: Optional[ProcessGroup] = None) ->
         backend._set_default_timeout(timeout)
 
 
+@_exception_logger
+@_time_logger
+def init(
+    world_size: int = -1,
+    rank: int = -1,
+    store: Optional[Store] = None,
+) -> None:
+    """
+    Initialize a distributed environment for current job. Once this API is
+    called, `get_world_size()` is guaranteed to return the number of processes
+    in current job, and `get_rank()` is guaranteed to return the rank of current
+    process in the job.
```

Comment:
Can the doc explain why you want to do this? (Namely, to skip PG initialization). Can we describe exactly how many comms this initialization will take?

Reply:
Thanks will add.